src/episerver_fmt_plug.c

```diff
@@ -134,6 +134,7 @@ static struct custom_salt {
 	int version;
 	unsigned char esalt[18 + 1]; /* base64 decoding, 24 / 4 * 3 = 18 */
 } *cur_salt;
+
 #if defined(_OPENMP) || defined(SIMD_COEF_32)
 static int omp_t = 1;
```

Comment:
Does this format need `omp_t` to be global to it? Why?

Reply:
Yes, it needs to be global. The `set_salt` is making use of `omp_t` variable.